website/docs/tutorials/06-deploying/README.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_label: Deployment
+---
+
+# Deployment
+
+## Prerequisites
+- [Factory Contract pattern](https://research.csiro.au/blockchainpatterns/general-patterns/contract-structural-patterns/factory-contract/)
```

Comment:
Does any part of this docs section use factory contract pattern or deploying a contract by another contract?

Reply:
Declaring doesn't make sense if you don't understand this pattern.

Reply:
Wouldn't be better to link StarkNet declaring docs instead some random resources?

Reply:
It should be StarkNet docs job to explain declaring, we should focus on explaining our interfaces